qutebrowser/config/configdata.yml

```diff
@@ -1235,6 +1235,11 @@ tabs.padding:
   type: Padding
   desc: Padding (in pixels) around text for tabs.
 
+tabs.persist_mode_on_change:
+  default: false
+  type: Bool
+  desc: When switching tabs, retain the current mode.
```

Comment:
For consistency with other boolean options, I'd vote to switch this around: `Retain the current mode when switching tabs.`

Reply:
Sounds good - I'll edit it.

Reply:
Done in 3b3acba34e99812532972e6fda6b392238f0d7f7